tests/lib.rs

```diff
@@ -133,16 +135,31 @@ impl VirtualFileSystem {
 
             println!("\n-- {} –-\n", file.display());
 
-            if let Some(extension) = file.extension() {
-                if extension == "tgz" {
-                    continue; // TODO(amello): make packaging deterministic so we can validate the files
+            match fs::read_to_string(&expected) {
+                Ok(expected_text) => {
+                    let actual_text = fs::read_to_string(&actual).expect("file not found");
+                    assert_str_eq!(expected_text, actual_text);
                 }
+                Err(err) if matches!(err.kind(), ErrorKind::InvalidData) => {
+                    // binary file, use checksum comparison
+
+                    fn hash_file(path: impl AsRef<Path>) -> String {
```

Comment:
can we use ring for the hash? there is a feature branch that also relies on ring and it would be nice to dedup dependencies

Reply:
Missed that dependency, will switch over.